receiver/awsxrayreceiver/README.md

```diff
@@ -1,5 +1,11 @@
 # AWS X-Ray Receiver
 
+| Status                   |           |
+| ------------------------ |-----------|
+| Stability                | [beta]    |
+| Supported pipeline types | traces    |
+| Distributions            | [contrib] |
```

Comment:
@willarmiros should the AWS distribution be included here?

Reply:
Yes please! https://aws-otel.github.io/docs/components/x-ray-receiver

Reply:
@TylerHelmuth @willarmiros added.